mopidy/audio/actor.py

```diff
@@ -574,6 +582,9 @@ def _on_source_setup(self, element, source):
         if self._live_stream and hasattr(source.props, "is_live"):
             gst_logger.debug("Enabling live stream mode")
             source.set_live(True)
+        else:
+            gst_logger.debug("Disabling live stream mode")
+            source.set_live(False)
```

Comment:
I can confirm there is no issue here and no requirement to disable live mode like this. Freshly created sources have their default value of live mode and we should not meddle with that.

Reply:
However... this does highlight a similar theoretical issue. If there was a type of source that did have live mode enabled by default then we would not realise and we would not ignore the buffering messages like we should. So technically we probably should sync our `self._live_stream` flag with `source.is_live`. To be honest I doubt there are any source elements like that because it's just asking for trouble.

Anyway, this is all out of scope here. The live mode changes need to be removed from this PR.